package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "splash-screen",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.2.1",
+    "node-sass": "^7.0.3",
+    "react": "^18.2.0",
+    "react-bootstrap": "^2.5.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
```

Comment:
Do we really need each one of these dependencies? I was thinking since we don't have a complex UI use case we could implement this using bare-minimum external dependencies. Totally fine if it is not possible, but just a thought to analyze each dependency individually as it will also improve future maintainability.

Reply:
👍 - we have to keep all of these dependencies updated as vulnerabilities are discovered, as well as keep the license info up to date.

we can follow the pattern here:
https://github.com/DynamoDS/NotificationCenter/pull/25 (thats fine in another PR).

Reply:
is web-vitals a peer dep of something? I don't see it used.
https://web.dev/vitals/

Reply:
Hi guys, thanks for noticing that. I removed the unused packages and I guess we can update the license in another PR.